.storybook/components/DesignTokens/Tier1/Colors.tsx

```diff
@@ -29,9 +29,6 @@ export const Tier1Colors = () => {
           listItems={getListItems('eds-color-other-yellow', 'yellow')}
         />
         <ColorList listItems={getListItems('eds-color-other-ruby', 'ruby')} />
-        <ColorList
```

Comment:
Removing as this token breaks the convention. Hiding the docs in preparation for removal